src/dktapps/pmforms/FormIcon.php

```diff
@@ -52,6 +52,7 @@ public function getData() : string{
 		return $this->data;
 	}
 
+	#[\ReturnTypeWillChange]
 	public function jsonSerialize(){
```

Comment:
If bc break is not a concern, it can be resolved by editing the return value
```suggestion
	public function jsonSerialize() : array{
```

Reply:
Realistically speaking no one should be overriding these methods (or extending the class at all), but that's not a given, so it has to remain this way for now.